src/content/docs/journey-migration/migrating-apm.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Install APM agents"
+metaDescription: "How to setup APM monitoring agents in New Relic."
+---
+
+import journeyAPMOverview from 'images/apm_screenshot-crop_overview.png'
+
+import journeyAPMPrefabAlerts from 'images/apm_screenshot-crop_precon-alerts.png'
+
+import journeyAPMDash from 'images/apm_screenshot-crop_journey-dash.png'
+
+import journeyAPMDash2 from 'images/apm_screenshot-crop_journey-dash-two.png'
+
+While setting up infrastructure monitoring provides a great foundation for understanding your systems, the best way to truly know your data comes from monitoring application performance. New Relic APM provides extensive, unified monitoring for all your applications. 
+
+That means we can better link your data through your apps and integrations, and provide more focused information for you through curated dashboards. That way, you get the level of depth and detail you need without having to hunt for what you want. And you can try out our extensive APM capabilities [absolutely free](https://newrelic.com/signup).
+
+<img
+    title="APM overview"
+    alt="An image displaying New Relic's APM monitoring page"
+    src={journeyAPMDash}
+/>
+
+## Your APM solution [#apm-solution]
+
+You may be used to `integrations` in Datadog, but in New Relic, we call these capabilities APM `agents`. We've set up our agents for both convenience and functionality. At New Relic, our agents are focused around language and linking the data they collect to other entities you've installed. 
```

Comment:
"Convenience and functionality" is a bit vague, what about "to be easy to install, but also to collect deep data about your app's performance and broad context about related entities." That's too long but hopefully makes sense directionally.